cmd/guide.go

```diff
@@ -48,176 +52,181 @@ you want to save details that don't fit in a single line.
 		},
 		{
 			"domain: task state",
-			`A task can be in one of two states: active or archived.
+			fmt.Sprintf(`A task can be in one of two states: **active** or **archived**.
 
 This list shows active tasks.
 
 To be pedantic about things, only the tasks in the active list are supposed to
 be "on your mind". However, there are benefits to having a list of archived
 tasks as well.
 
-Press <tab> to see the archived list.
-`,
+Press %s to see the archived list.
+`, "`<tab>`"),
 			true,
 		},
 		{
 			"domain: task details",
-			`The "Task Details" pane is intended for when you simply want to read all the
+			fmt.Sprintf(`The **Task Details** pane is intended for when you simply want to read all the
 details associated with a task in a full screen view.
 
-You can view this pane by pressing "d".
+You can view this pane by pressing %s. This pane is useful when a task's
+context is too long to fit in the context pane.
 
 Whilst in this pane, you can move backwards and forwards in the
-task list by pressing "h/<-/->/l". You quit out of this pane by either pressing
-"d" again, or q/esc/ctrl+c.
+task list by pressing %s. You quit out of this pane by either pressing
+%s again, or %s.
 
 Try it out. Come back to this entry when you're done.
-`,
+`, "`d`", "`h/<-/->/l`", "`d`", "`q/esc/<ctrl+c>`"),
```

Comment:
You could use an Unicode character maybe

https://en.m.wikipedia.org/wiki/Arrow_(symbol)#Unicode


```suggestion
`, "`d`", "`h/←/→/l`", "`d`", "`q/esc/<ctrl+c>`"),
```

I'm suggesting this because you are already using an Unicode character for the enter key